pkgs/development/python-modules/aioredis/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv
+, fetchPypi
+, buildPythonPackage
+, hiredis, async-timeout
+}:
+buildPythonPackage rec {
+
+  pname = "aioredis";
+  version = "1.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "84d62be729beb87118cf126c20b0e3f52d7a42bb7373dc5bcdd874f26f1f251a";
+  };
+
+  propagatedBuildInputs = [
+    hiredis
```

Comment:
You need to add this conditioned on `!isPyPy` https://github.com/aio-libs/aioredis/blob/ba9f5b707a29df5c90bc0a1c242d835564407894/setup.py#L9